### PR TITLE
Move Web template to csproj

### PR DESCRIPTION
**Pull request by piotrpMSFT**
Move Web template to csproj

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Windows_NT x86 Debug Build please.

**Comment by piotrpMSFT** [comment]
@mlorbetske can you take a look and give me a :shipit:?

@eerhardt @brthor @jgoshi @livarcocc  can one of you take a look as well?

File: src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Temporarily simulate an inner build until cross-targeting support for VS is checked in. This needs to be
+       above the import of the props because the SDK.props will only get imported if the TargetFramework is set.-->
+  <PropertyGroup>
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj]: I think this file name should be $projectname$.csproj

File: src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    <Compile Include="**\*.cs" Exclude="$(GlobalExclude)" />
+    <EmbeddedResource Include="**\*.resx" Exclude="$(GlobalExclude)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.App">
+      <Version>1.0.1</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.Web">
+      <Version>1.0.0-alpha-20160925-2</Version>
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj]: This is pretty old.  Is there a newer version that can be used?

File: src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions">
+      <Version>1.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink.Loader">
+      <Version>14.0.0</Version>
+    </PackageReference>
+  </ItemGroup>
+
+
+  <!--
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj]: ???

File: src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    <PackageReference Include="Microsoft.Extensions.Logging.Console">
+      <Version>1.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug">
+      <Version>1.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions">
+      <Version>1.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink.Loader">
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/CSharp_Web/MvcApp.csproj]: ???

**Comment by piotrpMSFT** [comment]
@mlorbetske please look at @eerhardt's feedback on the template itself. I will rename the .csproj to $projectname$.csproj to allow dotnet-new to rename the file to match the containing directory. Other aspects came directly from the template you shared.

**Comment by mlorbetske** [comment]
@phenning this content doesn't look right - all the links are http now instead of https, can you take a look at this?

**Comment by piotrpMSFT** [comment]
@srivatsn I know you're blocking on this change, so FYI. 

@mlorbetske @phenning I'm happy to take a different template. Please send my way.

**Comment by phenning** [comment]
:shipit:

**Comment by piotrpMSFT** [comment]
@morrisjoe @srivatsn once this is merged we will have updated Web templates

File: src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="**\*.cs" Exclude="$(GlobalExclude)" />
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-new/CSharp_Web/$projectName$.csproj]: Do you need the Excludes? I thought they were default now.

**Pull request merged**